duckduckgo/Casper
# Visual Upgrades for the Blog

## Font Changes

We're now using Proxima Nova for headers and Athelas for the content.

<img width="1728" alt="screen shot 2019-01-04 at 11 51 08 am" src="https://user-images.githubusercontent.com/81969/50699876-0a9f1200-1017-11e9-8e0b-aca9701dc16b.png">

<img width="1728" alt="screen shot 2019-01-04 at 11 51 06 am" src="https://user-images.githubusercontent.com/81969/50699877-0a9f1200-1017-11e9-9cc9-e06d11b5397d.png">

## Blue Branding Color

We're now using these colors for branding:

```css
    --cornflower-blue: #678fff;
    --cornflower-blue-dark: #3F66D5;
```

<img width="1728" alt="screen shot 2019-01-04 at 11 53 03 am" src="https://user-images.githubusercontent.com/81969/50699955-4fc34400-1017-11e9-90c4-341307190d00.png">

<img width="1728" alt="screen shot 2019-01-04 at 11 53 31 am" src="https://user-images.githubusercontent.com/81969/50699983-623d7d80-1017-11e9-8c67-6da40f5d181e.png">

## Add support for full-width images

We can now set full-width images in the blog.

<img width="1728" alt="screen shot 2019-01-04 at 11 54 40 am" src="https://user-images.githubusercontent.com/81969/50700047-8d27d180-1017-11e9-8e9d-1e2a23088d26.png">

## How to test:

1. Make sure you have ghost v1. Install using `ghost install local --v1`
2. Delete the image directory at `content/images` by running `rm -r content/images`
2. Use https://cl.ly/c6aeacf41a98/download/images.zip and extract to `content/images` by running `zip images.zip`
3. Delete all the content in your Ghost instance.

<img width="1728" alt="screen shot 2019-01-04 at 11 57 40 am" src="https://user-images.githubusercontent.com/81969/50700208-f7407680-1017-11e9-9ab3-8540caea1cc4.png">

4. Replace it with https://cl.ly/78fb044067e2/download/duckduckgo-blog.ghost.2019-01-04.json
5. You should now have parity with https://spreadprivacy.com/. 
6. See if there are any broken blog posts.
